### PR TITLE
Add foundry with its Fixtures

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

Add foundry and its fixtures.

### CODE REVIEW

File: backend/config/packages/dev/zenstruck_foundry.yaml [hunk 1]
```diff
@@ -5,12 +5,5 @@
 # For the full copyright and license information, please view the LICENSE
 # file that was distributed with this source code.
 
-nelmio_alice:
```
Comment (jordisala1991) [inline backend/config/packages/dev/zenstruck_foundry.yaml:8]: This change should be reverted.

File: backend/src/Fixtures/UserFactory.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            'plainPassword' => self::faker()->word(),
+            'privacyPolicy' => self::faker()->boolean(),
+            'allowShareData' => self::faker()->boolean(),
+            'active' => self::faker()->boolean(),
+            'linkedinProfile' => self::faker()->boolean(),
+            'twitterProfile' => self::faker()->boolean(),
+            'locale' => self::faker()->languageCode(),
+            'createdAt' => self::faker()->dateTime(),
+            'updatedAt' => self::faker()->dateTime(),
+            'fishbowls' => [FishbowlFactory::new()],
```
Comment (jordisala1991) [inline backend/src/Fixtures/UserFactory.php:38]: I think we can remove this property

File: backend/src/Fixtures/SonataUserUserFactory.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+final class SonataUserUserFactory extends ModelFactory
+{
+    protected function getDefaults(): array
+    {
+        return [
+            'username' => self::faker()->unique()->userName(),
+            'email' => self::faker()->unique()->email(),
+            'password' => self::faker()->password(),
+            'enabled' => self::faker()->boolean,
```
Comment (jordisala1991) [inline backend/src/Fixtures/SonataUserUserFactory.php:27]: boolean is a function()

File: backend/src/Fixtures/SonataUserUserFactory.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            'roles' => [],
```
Comment (jordisala1991) [inline backend/src/Fixtures/SonataUserUserFactory.php:28]: roles should be removed if it left blank

File: backend/src/Fixtures/UserFactory.php (hunk 2 above)
Comment (aerrasti) [reply backend/src/Fixtures/UserFactory.php]: By default not creating a fishbowl. ok.

**Review by aerrasti — APPROVED** [review]

**Pull request merged**